includes/class-events.php

```diff
@@ -559,6 +559,99 @@ public function update_run_status( $new_status ) {
 
 		return update_option( self::DISABLE_RUN_OPTION, $new_status );
 	}
+
+	/**
+	 * Query for multiple events.
+	 *
+	 * @param array $query_args Event query args.
+	 * @return array An array of Event objects.
+	 */
+	public static function query( array $query_args = [] ): array {
+		$events  = Events_Store::_query_events_raw( $query_args );
```

Comment:
Could we rename to `$event_db_rows` to make it clear that is NOT a collection ov Event objects?